lib/get-all-textpages.ts

```diff
@@ -0,0 +1,17 @@
+import hygraphClient, { gql } from '@/lib/hygraph-client'
+
+export const getAllTextPagesQuery = gql`
+  query AllTextPagesQuery() {
+    pages() {
+      slug
+      title
+    }
+  }
+`
+
+export const getAllTextPages = async (): Promise<any> => {
```

Comment:
lets remove `any` and create a proper interface that will describe the response we return as `pages`

Reply:
What is the correct way? Other pages (products, collections) use a single generic type only.

1. Add a special interface that only contains keys/values used in paths

`interface PagePath {
  slug: string
  title: string
}`

2. Use a more generic type:

`interface TextPage {
  id: string
  name: string
  type: string
  slug: string
  title: string
  page: {
    content: {
      html: string
    }
  }
}`

Reply:
interface TextPage seems what we need!